packages/data/src/useWalkthroughTestData.ts

```diff
@@ -29,6 +29,23 @@ export function getMultiChoiceQuestion() {
   };
 }
 
+export function getQuestion(questionId: string) {
+  // find the question for the given questionId
+  const questions: {
+    [key: string]: QuestionDisplayData | QuestionVariableData;
+  } = testCase999.questions;
```

Comment:
Will this only work for Workflow 1?

Reply:
Technically yes but since that file is only used essentially as a "test" helper to test the handling of various types of questions it's ok since it doesn't actually matter what workflow it is here.